sensor/kubernetes/upgrade/command_handler_impl.go

```diff
@@ -157,6 +158,19 @@ func (h *commandHandler) ProcessMessage(msg *central.MsgToSensor) error {
 	return nil
 }
 
+// upgradesSupported nil if upgrades are supported, otherwise it returns an error with a reason.
+func upgradesSupported(confH config.Handler) error {
+	if confH.GetHelmManagedConfig() != nil {
+		switch confH.GetHelmManagedConfig().GetManagedBy() {
```

Comment:
Optional nitpick: it might be better to pass the smallest needed thing, rather than the whole interface.

```suggestion
func upgradesSupported(helmManagedConfig *central.HelmManagedConfigInit) error {
	if helmManagedConfig != nil {
		switch helmManagedConfig.GetManagedBy() {
```